SharedPackages/BrowserServicesKit/Sources/Subscription/SubscriptionUserScript/SubscriptionUserScript.swift

```diff
@@ -30,21 +30,52 @@ protocol SubscriptionUserScriptHandling {
 
     /// Returns the details of Privacy Pro subscription.
     func subscriptionDetails(params: Any, message: UserScriptMessage) async throws -> SubscriptionUserScript.DataModel.SubscriptionDetails
+
+    // Returns the AuthToken of the subscription.
+    func getAuthAccessToken(params: Any, message: any UserScriptMessage) async throws -> SubscriptionUserScript.DataModel.GetAuthAccessTokenResponse
```

Comment:
Minor - As on line 60 below we could define `typealias DataModel = SubscriptionUserScript.DataModel` in this protocol and use it here to make these definitions (the return type) more readable